Description: Set the library version of libsbsms to get the correct .so name.
Author: Benjamin Drung <bdrung@debian.org>

Index: libsbsms/src/Makefile.am
===================================================================
--- libsbsms.orig/src/Makefile.am
+++ libsbsms/src/Makefile.am
@@ -32,3 +32,4 @@ libsbsms_la_SOURCES = \
 	subband.h \
 	track.h \
 	$(NULL)
+libsbsms_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@
